polyfill/test/duration.mjs

```diff
@@ -24,59 +24,11 @@ describe('Duration', () => {
-      equal(d3.toString({ smallestUnit: 'milliseconds' }), d3.toString({ fractionalSecondDigits: 3 }));
-      equal(d3.toString({ smallestUnit: 'microseconds' }), d3.toString({ fractionalSecondDigits: 6 }));
-      equal(d3.toString({ smallestUnit: 'nanoseconds' }), d3.toString({ fractionalSecondDigits: 9 }));
-    });
-    it('truncates or pads to 2 places', () => {
-      const options = { fractionalSecondDigits: 2 };
-      equal(d1.toString(options), 'PT15H23M0.00S');
-      equal(d2.toString(options), 'PT15H23M30.00S');
-      equal(d3.toString(options), 'PT15H23M30.54S');
-    });
-    it('pads to 7 places', () => {
-      const options = { fractionalSecondDigits: 7 };
-      equal(d1.toString(options), 'PT15H23M0.0000000S');
-      equal(d2.toString(options), 'PT15H23M30.0000000S');
-      equal(d3.toString(options), 'PT15H23M30.5432000S');
-    });
-    it('auto is the default', () => {
-      [d1, d2, d3].forEach((d) => equal(d.toString({ fractionalSecondDigits: 'auto' }), d.toString()));
-    });
-    it('accepts and truncates fractional fractionalSecondDigits', () => {
-      equal(d3.toString({ fractionalSecondDigits: 5.5 }), 'PT15H23M30.54320S');
-    });
-    it('smallestUnit overrides fractionalSecondDigits', () => {
-      equal(d3.toString({ smallestUnit: 'seconds', fractionalSecondDigits: 9 }), 'PT15H23M30S');
-    });
-    it('rounds to nearest', () => {
-      equal(d3.toString({ smallestUnit: 'seconds', roundingMode: 'halfExpand' }), 'PT15H23M31S');
-      equal(d3.toString({ fractionalSecondDigits: 3, roundingMode: 'halfExpand' }), 'PT15H23M30.543S');
-    });
-    it('rounds up', () => {
-      equal(d3.toString({ smallestUnit: 'seconds', roundingMode: 'ceil' }), 'PT15H23M31S');
-      equal(d3.toString({ fractionalSecondDigits: 3, roundingMode: 'ceil' }), 'PT15H23M30.544S');
-    });
-    it('rounds down', () => {
-      equal(d3.negated().toString({ smallestUnit: 'seconds', roundingMode: 'floor' }), '-PT15H23M31S');
-      equal(d3.negated().toString({ fractionalSecondDigits: 3, roundingMode: 'floor' }), '-PT15H23M30.544S');
-    });
-    it('truncates', () => {
-      equal(d3.toString({ smallestUnit: 'seconds', roundingMode: 'trunc' }), 'PT15H23M30S');
-      equal(d3.toString({ fractionalSecondDigits: 3, roundingMode: 'trunc' }), 'PT15H23M30.543S');
-    });
     it('rounding can affect units up to seconds', () => {
       const d4 = Duration.from('P1Y1M1W1DT23H59M59.999999999S');
       equal(d4.toString({ fractionalSecondDigits: 8, roundingMode: 'halfExpand' }), 'P1Y1M1W1DT23H59M60.00000000S');
     });
-    it('options may only be an object or undefined', () => {
-      [null, 1, 'hello', true, Symbol('foo'), 1n].forEach((badOptions) =>
-        throws(() => d1.toString(badOptions), TypeError)
-      );
+    it('options may be an object or undefined', () => {
       [{}, () => {}, undefined].forEach((options) => equal(d1.toString(options), 'PT15H23M'));
     });
```

Comment:
```suggestion
```

Covered in `test/built-ins/Temporal/Duration/prototype/toString/{fractionalseconddigits-undefined.js,options-undefined.js}`